exampleCourse/questions/fibonacciEditor/question.html

```diff
@@ -1,22 +1,21 @@
 <pl-question-panel>
-  <p>
-    The Fibonacci numbers are 1, 1, 2, 3, 5, 8, ..., where each
-    number is the sum of the two before it.
-    \[
-    F_n = F_{n-1} + F_{n-2}
-    \]
-  </p>
-  <p>
-    Write a Python function <tt>fib</tt> that takes a number <tt>n</tt>
-    and returns the n<sup>th</sup> Fibonacci number.
-  </p>
-  <pl-file-editor file-name="fib.py" ace-mode="ace/mode/python">
-def fib(n):
-    pass
-  </pl-file-editor>
+<p>
+The Fibonacci numbers are 1, 1, 2, 3, 5, 8, ..., where each
+number is the sum of the two before it.
+\[
+F_n = F_{n-1} + F_{n-2}
+\]
+</p>
+<p>
+Write a Python function <tt>fib</tt> that takes a number <tt>n</tt>
+and returns the n<sup>th</sup> Fibonacci number.
+</p>
+
+<pl-file-editor file-name="fib.py" ace-mode="ace/mode/python" source-file-name="tests/initial_code.py"></pl-file-editor>
+
 </pl-question-panel>
 
 <pl-submission-panel>
-  <pl-file-preview></pl-file-preview>
-  <pl-external-grader-results></pl-external-grader-results>
+<pl-file-preview></pl-file-preview>
+<pl-external-grader-results></pl-external-grader-results>
```

Comment:
Most of the changes to this file seem to just be removing indenting. Was this intentional or did an editor do it? I'd rather change only the single <pl-file-editor> line that really needed changing, if possible.